modules/nw-ovn-ipsec-disable.adoc

```diff
@@ -10,23 +10,36 @@ As a cluster administrator, you can disable IPsec encryption.
 
 .Prerequisites
 
-* Install the OpenShift CLI (`oc`).
-* Log in to the cluster with a user with `cluster-admin` privileges.
+* You installed the {oc-first}.
+* You logged in to the cluster with a user with `cluster-admin` privileges.
 
 .Procedure
 
-. To disable IPsec encryption, enter the following command:
+. Choose one of the following options to disable IPsec encryption:
++
+.. Where the `ipsecConfig.mode` parameter is set to either `External` or `Full` and the `ipsecConfig.full` schema is not added to `networks.operator.openshift.io`, enter the following command:
 +
 [source,terminal]
 ----
-$ oc patch networks.operator.openshift.io cluster --type=merge \
--p '{
+$ oc patch networks.operator.openshift.io cluster --type=merge -p \
+  '{
   "spec":{
     "defaultNetwork":{
       "ovnKubernetesConfig":{
         "ipsecConfig":{
           "mode":"Disabled"
         }}}}}'
 ----
++
+.. Where the `ipsecConfig.mode` parameter is set `Full` and the `ipsecConfig.full` configuration is added to `networks.operator.openshift.io`, enter the following command:
++
+[source,terminal]
+----
+$ oc patch networks.operator.openshift.io cluster --type='json' -p \
+      '[{"op": "remove", "path": "/spec/defaultNetwork/ovnKubernetesConfig/ipsecConfig/full"}, \ 
```

Comment:
Need remove ` \ ` at end of this line

The working command
```
% oc patch networks.operator.openshift.io cluster --type='json' -p \
      '[{"op": "remove", "path": "/spec/defaultNetwork/ovnKubernetesConfig/ipsecConfig/full"},  
      {"op": "replace", "path": "/spec/defaultNetwork/ovnKubernetesConfig/ipsecConfig/mode", "value": "Disabled"}]'
network.operator.openshift.io/cluster patched
```

Reply:
:heavy_check_mark: